samtranslator/model/__init__.py

```diff
@@ -350,6 +350,8 @@ def validate_properties_and_return_model(self, cls: Type[RT]) -> RT:
             with suppress(KeyError):
                 error_properties = ".".join(str(x) for x in e.errors()[0]["loc"])
             raise InvalidResourceException(self.logical_id, f"Property '{error_properties}' is invalid.") from e
+        except AttributeError as e:
+            raise InvalidResourceException(self.logical_id, str(e)) from e
```

Comment:
What you print now is the attribute error not original error. Can we include the original error msg?